packages/react/src/components/cookieConsentCore/translations.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-param-reassign */
+
+/**
+ * Retrieves a translation based on the provided key, language, and parameters.
+ *
+ * @param {object} translations - The translations object.
+ * @param {string} key - The translation key.
+ * @param {string} lang - The language code.
+ * @param {string} fallbackLang - The fallback language code in case the translation is missing in the provided language.
+ * @param {object} parameters - The parameters used to replace placeholders in the translation.
+ * @return {string} - The translated string.
+ * @throws {Error} - If the translation is missing for the provided key and language.
+ */
+// eslint-disable-next-line import/prefer-default-export
+export function getTranslation(translations, key, lang, fallbackLang, parameters) {
+  // Debug mode, return key instead of translation
+  if (lang === 'key') {
```

Comment:
This debug mode is inteded for the people (developers) using the banner, they can choose to set the language to "key" instead of normal one like "fi" to have the banner show translation keys in place of the actual translations. This helps taking screenshots of the banner and explaining context to translators where their translations would appear.

So, this is a feature in our book.